networkx/algorithms/dag.py

```diff
@@ -725,6 +815,12 @@ def antichains(G, topo_order=None):
     NetworkXUnfeasible
         If `G` contains a cycle
 
+    Examples
```

Comment:
The example looks good, but I just noticed that the `Returns` section above should instead be a `Yields` since this is a generator, something like:

```
Yields
------
antichain : list
    list of nodes in `G` representing an antichain